programmers/난이도별/level01.두_정수_사이의_합/HyeonJeong.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+long long solution(int a, int b) {
+    long long answer = 0;
+
+    if(a>b)
+        for(int i = 0; i <= a - b; i++)
+            answer += b + i;
+    else if(a<b)
+        for(int i = 0; i <= b - a; i++)
+            answer += a + i;
+    else
+        answer = a;
+    return answer;
```

Comment:
```suggestion
long solution(int a, int b) {
    long answer = 0;
    int min = a < b ? a : b;
    int max = a > b ? a : b;
    for(int i = 0; i <= max - min; i++)
        answer += min + i;
    return answer;
}
```

이렇게 min/max 형태로 저장하면 더 짧고 직관적으로 풀이할 수 있습니다~

Reply:
@JunilHwang 현업에서도 삼항 연산자 자주 쓰시나요?

Reply:
@dkdlelk99 자주 사용합니다!

Reply:
@dkdlelk99 자주 사용합니다!